test/units/TEST-75-RESOLVED.sh

```diff
@@ -898,5 +785,169 @@ grep -qF "DNS Servers: 8.8.8.8" /tmp/output
 
 # Check if resolved exits cleanly.
 restart_resolved
+}
+
+# PRE-SETUP
+systemctl unmask systemd-resolved.service
+systemctl enable --now systemd-resolved.service
+systemctl service-log-level systemd-resolved.service debug
+
+# Need to be run before SETUP, otherwise things will break.
+manual_testcase_01_resolvectl
+manual_testcase_02_mdns_llmnr
+
+### SETUP ###
+echo "SETUP BEGIN" | systemd-cat -p info
```

Comment:
No need for the pipe, everything is running with `set -x` and is logged to journal. Just use the

```bash
: "Setup BEGIN"
```

syntax as it's used everywhere else in the file, to avoid duplicate echo. Also, you can drop the comment, as it's now unnecessary.